extensions/binding/org.eclipse.smarthome.binding.mqtt/src/main/java/org/eclipse/smarthome/binding/mqtt/handler/BrokerHandler.java

```diff
@@ -129,11 +126,12 @@ public void dispose() {
      * @throws IllegalArgumentException Throws this exception, if provided hash values cannot be
      *             assigned to the {@link PinningSSLContextProvider}.
      */
-    protected void assignSSLContextProvider(MqttBrokerConnection connection) throws IllegalArgumentException {
+    protected static void assignSSLContextProvider(BrokerHandlerConfig config, MqttBrokerConnection connection,
```

Comment:
My guess about this being `static` is the fact that no instance variables are involved. Maybe this should be moved to a util class or made none static again. wdyt?